SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJobWebRunner/BrokerProfileOptOutSubJobWebRunner.swift

```diff
@@ -110,11 +123,11 @@ final class OptOutJob: DataBrokerJob {
         }
     }
 
-    func extractedProfiles(profiles: [ExtractedProfile], meta: [String: Any]?) async {
+    public func extractedProfiles(profiles: [ExtractedProfile], meta: [String: Any]?) async {
```

Comment:
Do you know why this exists? What is meta?

Reply:
`meta` comes from CCF, but it doesn't seem to be directly used by any major parts of PIR, rather I only see it used for the debug scan (`DebugScanReturnValue`). It does feel like we'd benefit here either from a property rename, or at _least_ a comment explaining what this is used for.